lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -134,16 +143,22 @@ function dosomething_campaign_preprocess_common_vars(&$vars, &$wrapper) {
         ),
       ),
       'twitter' => array(
```

Comment:
@angaither a little confused here, but maybe I misunderstood the [issue](https://github.com/DoSomething/phoenix/issues/6584)? Is this correct? We want to give admins the ability to change social share text but if they don't customize it, we just populate the share language with the CTA.